### ISSUE TITLE
Fix configure error and warnings with cmake

### ISSUE THREAD

**Issue by Warchant** [hyperledger/iroha-ametsuchi#17]
```
$ mkdir build
$ cd build
$ cmake ..
... < ok > ...
CMake Warning (dev) at external/src/gtest/CMakeLists.txt:3 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at external/src/gtest/googletest/CMakeLists.txt:47 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
CMake Error in CMakeLists.txt:
  Imported target "keccak" includes non-existent path

    "/home/bogdan/tools/iroha-ametsuchi/external/src/keccak/bin/generic64/libkeccak.a.headers"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in CMakeLists.txt:
  Imported target "keccak" includes non-existent path

    "/home/bogdan/tools/iroha-ametsuchi/external/src/keccak/bin/generic64/libkeccak.a.headers"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.

```

**Comment by lebdron** [comment]
Policy issue is present in googletest https://github.com/google/googletest/issues/709

**Comment by Warchant** [comment]
closed by #20

**Comment by ikoruk** [comment]
While the dependencies have been fixed, the warnings are still present.
